ocamldoc/odoc_sig.ml

```diff
@@ -1211,6 +1212,18 @@ module Analyser =
               else
                 None
             in
+            let (maybe_more, info_after_opt) =
+              My_ir.just_after_special
+                !file_name
+                (get_string_of_file pos_end_ele pos_limit)
+            in
+            let comment_opt = merge_infos comment_opt info_after_opt in
+            let comment_opt = analyze_alerts comment_opt pmd_attributes in
+            let comment_opt =
+              match module_type.Parsetree.pmty_desc with
+              | Parsetree.Pmty_signature sg -> analyze_toplevel_alerts comment_opt sg
```

Comment:
Toplevel alerts are only recognized by the compiler at the compilation-unit level:
```ocaml
module M :sig [@@@deprecated] end
```
does not attach the alert to `M` currently.
This is already slightly confusing, I would rather not have `ocamldoc` add more confusion here.

Reply:
I've pushed a fix.